src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs

```diff
@@ -1435,6 +1435,9 @@ public partial class DelegatingConfigHost : System.Configuration.Internal.IInter
         public virtual void VerifyDefinitionAllowed(string configPath, System.Configuration.ConfigurationAllowDefinition allowDefinition, System.Configuration.ConfigurationAllowExeDefinition allowExeDefinition, System.Configuration.Internal.IConfigErrorInfo errorInfo) { }
         public virtual void WriteCompleted(string streamName, bool success, object writeContext) { }
         public virtual void WriteCompleted(string streamName, bool success, object writeContext, bool assertPermissions) { }
+#if netfx
```

Comment:
We should not have any ifef'ed code here in the ref. They should either be removed completely or implemented in the core implementation.